app/src/contexts/Options/Options.tsx

```diff
@@ -130,33 +138,52 @@ const Options: React.FC = (props) => {
               registry.interface.parseLog(log).args.optionAddress,
               provider
             )
-            if (option.optionParameters.base.asset.symbol !== assetName) return
-            const { premium, reserve } = await getPairData(
-              provider,
-              option.address
-            )
-            pairReserveTotal += reserve
-            if (!option.isCall) {
+            let baseAssetSymbol = option.optionParameters.base.asset.symbol
+            switch (assetName.toLowerCase()) {
```

Comment:
at least we only need this for eth, in the future we need our market IDs to be the same as the contract ERC20 symbol to keep things clean

Reply:
Was a quick fix, I'll link it up with the coin gecko id constant